app/allocations/middleware.js

```diff
@@ -0,0 +1,51 @@
+const { format } = require('date-fns')
+const allocationService = require('../../common/services/allocation')
+
+const {
+  dateFormat,
+  getDateFromParams,
+  getPeriod,
+} = require('../../common/helpers/date-utils')
+
+const allocationsMiddleware = {
+  // todo: refactor this to be common with the /moves pagination.
```

Comment:
I believe if you use capitals as the syntax some code editors will highlight the todo (well, mine does at least) to make them easier to spot (and hopefully address):

![image](https://user-images.githubusercontent.com/3327997/79572890-95453c80-80b5-11ea-976f-6d74f8ed4a90.png)

![image](https://user-images.githubusercontent.com/3327997/79572905-9ecea480-80b5-11ea-9ca6-209afe6af0d7.png)

Reply:
Ha! Webstorm does it with both cases 😄 But happy to capitalise it if it's the standard.